0TA_Template_Sketch/credentials.h

```diff
@@ -0,0 +1,2 @@
+#define mySSID      "enter_your_wifi_network_name_here"
+#define myPASSWORD  "enter_your_wifi_network_password_here"
```

Comment:
Define is a pre-processor command. So it would be better to declare constant variables here. See this [link](https://www.includehelp.com/cpp-tutorial/difference-between-const-and-define.aspx) for a more in depth explanation. Also you did forget ``#pragma once`` or equivalent. Other than that you did a great job. I did check with the code in my PR and it's working since you did contact the developper of TelnetStream, so you can close your PR (we are providing more-or-less the same changes and my PR is a little bit ahead in terms of implemented features since I've updated both projects, etc.). Thanks for your help in solving the Telnet issue.